### ISSUE TITLE
DOC: kwarg `persist`

### ISSUE THREAD

**Issue by michaelaye** [holoviz/hvplot#112]
Hi! I'm trying out hvplot and working through the `Plotting` guide here:
https://hvplot.pyviz.org/user_guide/Plotting.html
and would like to know what `persist` does as it isn't described.

I tried df.hvplot? but that doesn't provide any docstring.

**Comment by ahuang11** [comment]
I think `persist` is loading the whole dataframe/dataset into memory and reusing it each time across cores: https://github.com/pydata/xarray/issues/1344 and http://xarray.pydata.org/en/stable/generated/xarray.Dataset.persist.html

**Comment by philippjfr** [comment]
Yup, @ahuang11 is correct, it's a dask specific option to persist the data to memory. I think it would make a lot of sense to have a page dedicated to describing all the options grouped into sensible chunks. We could also consider putting a lot of it into the docstring but they might get somewhat unwieldy. 

>I tried df.hvplot? but that doesn't provide any docstring.

That will be fixed in the next release.
